tests/Order/Customer/LocaleTest.php

```diff
@@ -31,7 +31,10 @@ public function itCanBeEmptyString()
     public function itShouldFailOnIncorrectLocale()
     {
         $this->setExpectedException('Assert\InvalidArgumentException');
-        Locale::fromString('en_US');
+        Locale::fromString('en');
```

Comment:
And since 'en' is valid, its test should look similar to `itShouldInstantiateFromAValidString`.